src/styles/themes/_dark-theme.scss

```diff
@@ -0,0 +1,1094 @@
+body[dark] {
```

Comment:
`:root` is effectively `html` but with slightly greater specificity. This is going to have slightly greater specificity. Not sure that is a huge concern but maybe?